src/MMLib.SwaggerForOcelot/DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -92,5 +95,16 @@ private static void AddAggregatesDocs(SwaggerGenOptions c, OcelotSwaggerGenOptio
                 c.DocumentFilter<AggregatesDocumentFilter>();
             }
         }
+
+        private static void IncludeXmlComments(string[] paths, SwaggerGenOptions c)
+        {
+            foreach (var path in paths)
```

Comment:
Please check if `paths` is not `null`.

Reply:
I made changes according to your comment using the new "is not" operator.